### PR TITLE
Adding default (or invalid) theme colors

### LINKED ISSUE #444: Default theme layout
We need a default theme layout other than a transparent one.
http://devv3.geoportail.lu/renaud/theme/test

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #444.

To test it, open the developer tools and remove the value for the `data-theme` attribute in `body` tag.

### CODE REVIEW

File: geoportailv3/static/less/theme.less [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+      background-color: white;
+    }
+    a:hover {
+      background-color: #ccc;
+    }
+    a:hover, a.active{
+      color: @primary;
+    }
+  }
+}
```
Comment (elemoine) [inline geoportailv3/static/less/theme.less]: I'd just add an empty line after this line for clarity.

**Comment by elemoine** [comment]
I just added a very minor comment, otherwise looks very good to me. This really demonstrates the benefits of using less!

**Comment by pgiraud** [comment]
Line added and indentation fixed (which makes a big diff).
Waiting for travis before merging.

**Comment by pgiraud** [comment]
Ignoring travis. Merging.

**Pull request merged**